cookbooks/aws-parallelcluster-common/resources/dns_domain/dns_domain.rb

```diff
@@ -16,6 +16,13 @@
 use 'partial/_dns_search_domain_ubuntu'
 use 'partial/_dns_search_domain_redhat'
 
+action :setup do
```

Comment:
let's change the default_action to be `:setup`, in this way is consistent with most of the other resources (and be sure we're calling the righ action in the two places).